great_expectations/cli/store.py

```diff
@@ -19,18 +19,35 @@ def store(ctx):
     # TODO consider moving this all the way up in to the CLIState constructor
     ctx.obj.data_context = context
 
+    usage_stats_prefix = f"cli.store.{ctx.invoked_subcommand}"
+    toolkit.send_usage_message(
+        data_context=context,
+        event=f"{usage_stats_prefix}.begin",
+        success=True,
+    )
+    ctx.obj.usage_event_end = f"{usage_stats_prefix}.end"
+
 
 @store.command(name="list")
 @click.pass_context
 def store_list(ctx):
     """List active Stores."""
     context = ctx.obj.data_context
-    stores = context.list_active_stores()
-    cli_message(f"{len(stores)} active Stores found:")
-    for store in stores:
-        cli_message("")
-        cli_message_dict(store)
+    usage_event_end: str = ctx.obj.usage_event_end
```

Comment:
Where is the `begin` event?..

Reply:
The begin events are all in the base commands! That way we never forget to add them for new CLI commands :) And also a bit more DRY. E.g. look at `def store()`